docker/dev/docker.env

```diff
@@ -32,3 +32,6 @@ EKS_NODEGROUP_ROLE_ARN=x
 C_FORCE_ROOT=true
 ENVIRONMENT=dev
 SERVICE_DNS=localhost
+
+GF_SECURITY_ADMIN_USER=evalai
+GF_SECURITY_ADMIN_PASSWORD=evalaiai
```

Comment:
@burnerlee can we change the username to `admin` and password to `password` for dev